apps/cookbook/src/app/examples/chart-wip-example/examples/chart-wip-example-column-stacked.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Component } from '@angular/core';
+import { ChartOptions } from 'chart.js';
```

Comment:
Should we ideally expose this through Kirby? 

We ended up doing a 'proxy' of the date-fns API for some properties in the Calendar component when it was reimplemented recently, and I like that approach as it shows what properties we _intend_ consumers to modify. And we can still reference the docs for the chart.js API directly, as it is 1 to 1.

So, something similar to [line 44](https://github.com/kirbydesign/designsystem/blob/master/libs/designsystem/src/lib/components/calendar/calendar.component.ts#L44) and [line 29](https://github.com/kirbydesign/designsystem/blob/master/libs/designsystem/src/lib/components/calendar/calendar.component.ts#L29), where we export a the Locale from date-fns so it can be referenced through Kirby.

Reply:
That would be much prettier - I agree! 
Will get that done.